schemas/session-replay/mobile/incremental-snapshot-record-schema.json

```diff
@@ -21,10 +21,7 @@
                     "readOnly": true
                 },
                 "data": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "incremental-data-schema.json"
-                    },
+                    "$ref": "incremental-data-schema.json",
                     "description": "The IncrementalData contained by this Record.",
                     "readOnly": true
```

Comment:
Won't `description` clash with what is defined in `$ref`'s `description`?

Reply:
Yes, but there's a merging mechanism, the `@ref` one takes priority.